dotnet-interactive/HttpApiBootstrapper.cs

```diff
@@ -9,12 +9,17 @@ namespace Microsoft.DotNet.Interactive.App
 {
     internal static class HttpApiBootstrapper
     {
-        public static string GetJSCode(Uri[] probingUris, string seed)
+        public static string GetHtmlInjection(Uri[] probingUris, string seed)
         {
             var apiCacheBuster = $"{Process.GetCurrentProcess().Id}.{seed}";
-            var template = @"// ensure `requirejs` is available
-if ((typeof (requirejs) !== typeof (Function)) || (typeof (requirejs.config) !== typeof (Function))) 
-{
+            var template = @"
+<div>
+    <div id='dotnet-interactive-this-cell-$SEED$' style='display: none'>
```

Comment:
this value is stored in the notebook as output, when a saved notebook loads this will be evaluated, then on the first run  a new div with new id will be added to the dom.  what will happen?

Reply:
Once this cell is re-evaluated, it's output will be overwritten.  Should be harmless.